tests/cases/fourslash/completionPreferredSuggestions1.ts

```diff
@@ -16,5 +16,5 @@
 verify.completions({ marker: "1", includes: ["a", "b", "c"] });
 verify.completions({ marker: "2", includes: ["0", "1", "2"], isNewIdentifierLocation: true });
 verify.completions({ marker: "3", includes: ["a", "b", "c"] });
-verify.completions({ marker: "4", excludes: ["a", "b", "c"] });
+verify.completions({ marker: "4", exact: [] });

```

Comment:
This case wasn't actually returning any meaningful completions, see the playground [here](https://www.typescriptlang.org/play?ts=5.0.0-dev.20230220#code/C4TwDgpgBAMglsCAnAhgGwKoDs4HssCMAPACpQQAeiWAJgM5QYA0jAfFALxRkA+jUAMigBvAL4BuAFA0IAYzQok0NBGBQAbgBYAXLATJ02PISIAiFKah9TAI0vXZplnWBI4WAOaspWzlFMA9ABUmkEBplJAA). It's just that **empty** `actualCompletions` were received [here](https://github.com/microsoft/TypeScript/blob/c6fb2e10dcdf4839924a4288dae1cfd93b3c1c57/src/harness/fourslashImpl.ts#L906) but with the current change in this PR I just don't return any completions here. I strongly believe that both are OK, it's just that `excludes` in the `verify.completions` call imply that there should be some completions (even if empty). It's purely a test setup/helper logic and not something that affects conumers.